src/channel_internal.c

```diff
@@ -322,10 +369,36 @@ IMPLEMENT_MOCKABLE_FUNCTION(, CHANNEL_RESULT, channel_internal_pull, THANDLE(CHA
             CONTAINING_RECORD(channel_internal_ptr->op_list.Flink, CHANNEL_OP, anchor)->pull_callback != NULL
             )
         {
-            if (enqueue_operation(channel_internal, out_op_pull, pull_callback, pull_context, NULL, NULL, NULL) != 0)
+            if (enqueue_operation(
+                channel_internal,
+                out_op_pull,
+                correlation_id,
+                pull_callback,
+                pull_context,
+                NULL,
+                NULL,
+                NULL,
+                NULL) != 0
+            )
             {
                 /*Codes_SRS_CHANNEL_INTERNAL_43_023: [ If there are any failures, channel_internal_pull shall fail and return CHANNEL_RESULT_ERROR. ]*/
-                LogError("Failure in enqueue_operation(channel_internal_ptr, out_op_pull, pull_callback, pull_context, NULL, NULL, NULL)");
+                LogError("Failure in enqueue_operation("
+                    " channel_internal = %p"
```

Comment:
>  [](http://example.com/codeflow?start=21&length=1)

extra space in logging here.

Also these multiline LogErrors are geo-ification of c-util, can we keep this repo somewhat consistent about logging?

Here and everywhere.

Reply:
300 characters in a single line is the better apparently